contracts/liquidators/LiquidationAuction02.sol

```diff
@@ -66,6 +92,12 @@ contract LiquidationAuction02 is ReentrancyGuard {
             collateralInPosition
         );
 
+        // ensure that at least 1 wei of Curve LP is transferred to cdp owner
```

Comment:
What for?

Reply:
We need this to ensure `_checkpoint()` is called for cdp owner at the [wrapper contract](https://github.com/iamdefinitelyahuman/curve-dao-contracts/blob/feat-gauge-wrapper-usdp/contracts/gauges/LiquidityGaugeWrapperUnit.vy) on each liquidation